### PR TITLE
Harden docker-based test runner.

### PR DESCRIPTION
**Pull request by paulp**

### CODE REVIEW

**Comment by jdegoes** [comment]
@rabbitonweb Please review / merge this.

**Comment by rabbitonweb** [comment]
Missed it, on it

File: README.md [hunk 1]
```diff
@@ -57,22 +57,14 @@ This will lead to failures in the integration test project (`it`). The reason fo
 1. Use dockerized integration tests script
```
Comment (rabbitonweb) [inline README.md]: This line above should also be changed to "Alternative: dockerized integration tests script" for consistency

**Comment by rabbitonweb** [comment]
Small change in README & LGTM

**Comment by paulp** [comment]
@rabbitonweb I pared down the language a bit further.

**Comment by rabbitonweb** [comment]
LGTM, should I merge?

**Comment by paulp** [comment]
@rabbitonweb would you mind merging this?

**Pull request merged**